.gitlab/butte.yml

```diff
@@ -0,0 +1,35 @@
+##############################################################################
```

Comment:
What is butte?

Reply:
Smaller lassen-like system, thought it was just for LC testing?

Reply:
I really don’t have the history.
Umpire had host-config files for Butte, so I ported the capability. If there is no interest in running on Butte, let me know :).

Reply:
No big deal. I don't recall hearing about it. If we test on lassen that should be sufficient.